internal/compiler-bridge/src/main/scala/xsbt/ExtractUsedNames.scala

```diff
@@ -49,29 +52,39 @@ import Compat._
 class ExtractUsedNames[GlobalType <: CallbackGlobal](val global: GlobalType) extends Compat with ClassName with GlobalHelpers {
   import global._
 
-  def extract(unit: CompilationUnit): Iterable[(String, Iterable[String])] = {
+  def extract(unit: CompilationUnit): HashMap[String, ArrayList[String]] = {
```

Comment:
Yeah, the issue here is that in Scala 2.11 you don't have SAM types, so you cannot use Scala functions instead of the required input for `foreach`. Don't worry about this!